tools/tree-diff

```diff
@@ -84,7 +84,7 @@ def symlink_diff(name, dir_fd1, dir_fd2, path, differences):
 # pylint: disable=too-many-branches
 def diff_aux(dir_fd1, dir_fd2, path, report):
     entries1 = set()
-    with os.scandir(dir_fd1) as it:
+    with os.scandir(f"/proc/self/fd/{dir_fd1}") as it:
```

Comment:
I still don't really like having to do this, but since it lives in `tools/` I won't think about it :)

Reply:
Me neither, but the same approach is already used in the same file. I'm open to any suggestions of you have some 😉